### PR TITLE
Pagination: Next & Previous buttons on content pages

### PR DESCRIPTION
**Pull request by lovemecomputer**
a new PR of https://github.com/carbon-design-system/carbon-website/pull/371 with v10 compatible code :)

this is a WIP PR, updates incoming!

closes https://github.com/carbon-design-system/carbon-website/issues/305

### CODE REVIEW

**Comment by alisonjoseph** [comment]
is this ready for review or still WIP @lovemecomputer ?

**Comment by alisonjoseph** [comment]
This is looking great!! couple things

- [ ] Need to add to the homepage
- [ ] Need it to loop (so need a link on homepage to link back to the last page and from last page to homepage

I have some suggestions to update the grid code I'll add

File: src/components/NextPrevious/next-previous.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.next-previous-wrapper {
+  background: #282828;
+}
+
+.next-previous-grid-container {
```
Comment (alisonjoseph) [inline src/components/NextPrevious/next-previous.scss]: lets delete this if we don't need it

File: src/components/NextPrevious/next-previous.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  padding: 2em 0;
+  flex: 1;
+  text-decoration: none;
+}
+
+.next-previous-link--previous {
+  margin-right: 16px;
+}
+
+.next-previous-link--next {
```
Comment (alisonjoseph) [inline src/components/NextPrevious/next-previous.scss]: delete this if we don't need

File: src/components/NextPrevious/next-previous.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.target-page-direction {
+  display: block;
+  font-size: 14px;
```
Comment (alisonjoseph) [inline src/components/NextPrevious/next-previous.scss]: update to use typescale

**Comment by lovemecomputer** [comment]
@alisonjoseph thanks, i pushed a commit fixing these!

File: src/components/NextPrevious/next-previous.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  text-decoration: none;
+}
+
+.next-previous-link--previous {
+  margin-right: 16px;
+}
+
+.target-page-direction {
+  display: block;
+  font-size: rem(14px);
```
Comment (alisonjoseph) [inline src/components/NextPrevious/next-previous.scss]:
```suggestion
  @include type-style('body-short-01');
```

File: src/components/NextPrevious/next-previous.scss [hunk 5]
```diff
@@ -0,0 +1,42 @@
+.next-previous-wrapper {
+  background: #282828;
```
Comment (asudoh) [inline src/components/NextPrevious/next-previous.scss]: `$next-previous-wrapper-bg: $ibm-colors__gray--90`?

File: src/components/NextPrevious/next-previous.scss [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.next-previous-controls a {
+  color: #ffffff;
+  padding-left: 16px;
+  transition: 0.35s ease all;
+}
+
+.next-previous-controls a:hover {
+  background: #565656;
```
Comment (asudoh) [inline src/components/NextPrevious/next-previous.scss]: `$next-previous-controls-hover: $ibm-colors__gray--70`?

**Comment by lovemecomputer** [comment]
@alisonjoseph on it! @asudoh thanks! 

looking at these today

**Comment by lovemecomputer** [comment]
cool, review issues have been addressed and this component is re-ready for review!

**Pull request merged**